server/services/zigbee2mqtt/exposes/enumType.js

```diff
@@ -111,6 +125,20 @@ module.exports = {
         },
       },
     },
+    volume: {
+      feature: {
+        category: DEVICE_FEATURE_CATEGORIES.SIREN,
+        type: DEVICE_FEATURE_TYPES.SIREN.VOLUME,
+      },
+    },
+    melody: {
+      feature: {
+        category: DEVICE_FEATURE_CATEGORIES.SIREN,
+        type: DEVICE_FEATURE_TYPES.SIREN.MELODY,
+        min: 1,
+        max: 18,
```

Comment:
Why did you set min/max values? If this attributes are provided by z2m device, remove them from here.

Reply:
OK I'll remove it